Tests/RockLib.HealthChecks.Tests/HealthCheckRunnerTests.cs

```diff
@@ -184,7 +184,7 @@ public async Task RunAsyncRunsAllTheHealthChecksAndCallsTheHealthResponseCustomi
 
             var runner = new HealthCheckRunner(healthChecks, responseCustomizer: responseCustomizer);
 
-            var response = await runner.RunAsync().ConfigureAwait(false);
```

Comment:
We are changing these to true now?

Reply:
Oh these are tests.  Nm.

Reply:
With the update to xunit.runner.visualstudio, you are start getting errors about parallelization of tests. Nice feature bump that seems to be a breaking change🙃